src/js/tests/staff-form.spec.js

```diff
@@ -98,7 +141,74 @@ describe('<StaffForm>', () => {
                 '/confirm'
             );
         });
+
+        it(`dispatches ${IS_LOADING}`, () => {
+            expect(props.dispatch).to.have.been.calledWith(
+                { type: IS_LOADING, loading: true }
+            );
+        });
+    });
+
+    context('when confirm booking is clicked and booking returns an error', () => {
```

Comment:
@contrapunctus-xv Is this really an error response spec?  I merged anyway but just FYI.